apps/folderlaunch/settings.ts

```diff
@@ -0,0 +1,272 @@
+(function (back: Function) {
```

Comment:
We could use the [`SettingsFunc` type](https://github.com/espruino/BangleApps/blob/master/typescript/types/funcs.d.ts#L1-L1) for more accurate types (`() => void` instead of `any => any`), so it'd go from something like:

```typescript
(function (back: Function) { ... })
```

to

```typescript
(function (back) { ... }) satisfies SettingsFunc
```

Reply:
I like that. I think it's always good to use more specific types when possible.

(Github claims I can accept this and have it automatically committed with you getting credit. I'm trying to figure out how to do that.)

Reply:
Cool - I'm not sure if I have to have put it in as a suggestion, does this work?
```suggestion
(function (back) {
```

But then I'd also need another suggestion for the `satisfies` bit at the end, so maybe not!

Reply:
There are obvious buttons now (which are grayed out because things have changed since then), but yeah, I think you did it right.

Reply:
Gotya, sounds good :)